Dev

eti88 opened the pull request:
### Radio details page changes

- Moved social link mockup to radio header
- Radio header now take entire radio object as prop 
- Radio details moved to separate popup
- Genres of the radio moved to body
- Added track image to body
- Added support for component Genre.vue for radio details page

eti88 committed:
Merge pull request #1 from bitsongofficial/dev

Dev

eti88 committed:
Removed tabs from, moved details to popup and changed body

eti88 committed:
Renamed and moved qrcode wallet dialog to template

angelorc commented:
Before merge the PR, complete these steps:

- [ ] In the 3-vertical dots, add a dropdown menu, then add `Details` and when an user click on it open the dialog.
- [ ] Connect the endpoint, I want to test the radios before to merge the PR
- [ ] Disable by adding a comment on social links

I cannot figure out some function related to the genres component. Also I see that this component is imported in other pages but the component is never used.

eti88 committed:
added dropdown menu for 3-vertical dots with popup trigger on details item

eti88 committed:
connected radio details page to endpoint

eti88 committed:
commented social links on radio header

eti88 commented:
- [x] In the 3-vertical dots, add a dropdown menu, then add Details and when an user click on it open the dialog.
- [x] Connect the endpoint, I want to test the radios before to merge the PR
- [x] Disable by adding a comment on social links

Also :
- [x] Renamed and moved QRcode wallet dialog to template with vuex interaction

eti88 commented:
The Genres component was created on the assumption that the genres property of the radio object only return ids.  The creation hook compare the given list with all genres. But as written in [the api-radio issue](https://github.com/bitsongofficial/radio-api/issues/1) with the slugs the component will be changed

eti88 committed:
Renamed component from ProfileWallet to WalletOverview

eti88 committed:
replaced wallet address placeholders with vuext getter

eti88 committed:
moved most of env variable to corresponding vuex getter

eti88 committed:
Changed balance component and updated item structure

eti88 committed:
Revert "moved most of env variable to corresponding vuex getter"

This reverts commit db520d246db2899510c9975d1ac1ed1d60c937bd.

eti88 committed:
Revert "replaced wallet address placeholders with vuext getter"

This reverts commit ba9898f3bc47604650979377df3ad644916319e1.

eti88 committed:
clean unrelated commits for this pr

eti88 committed:
Revert "Renamed component from ProfileWallet to WalletOverview"

This reverts commit 542350233406f47aa5bf018f2dd2ae757025c357.

eti88 committed:
Revert "Changed balance component and updated item structure"

This reverts commit 3d2bdaddba8ad15198125f097e4acb68a12269bc.

eti88 committed:
Revert "Revert "moved most of env variable to corresponding vuex getter""

This reverts commit fb4bc68ad817c4275e80f7ee77599b1c8fae7ed9.

eti88 committed:
Revert "Revert "replaced wallet address placeholders with vuext getter""

This reverts commit 7477969417484cc4b6984a75ee8781ebbdf93456.

eti88 committed:
revert other unrelated commit

eti88 committed:
Revert "moved most of env variable to corresponding vuex getter"

This reverts commit db520d246db2899510c9975d1ac1ed1d60c937bd.

eti88 committed:
Revert "replaced wallet address placeholders with vuext getter"

This reverts commit ba9898f3bc47604650979377df3ad644916319e1.

eti88 committed:
Merge pull request #2 from bitsongofficial/dev

Dev
